src/main/java/az/ingress/strategy/concrete/ProductCreatedNotificationStrategy.java

```diff
@@ -0,0 +1,29 @@
+package az.ingress.strategy.concrete;
+
+import az.ingress.config.RabbitMQProperties;
+import az.ingress.model.dto.NotificationDto;
+import az.ingress.queue.sender.QueueSender;
+import az.ingress.strategy.abstraction.NotificationStrategy;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import static az.ingress.model.enums.ChannelType.TELEGRAM;
+
+@Component
+@RequiredArgsConstructor
+public class ProductCreatedNotificationStrategy implements NotificationStrategy {
+    private final QueueSender queueSender;
+    private final RabbitMQProperties rabbitMQProperties;
+    private static final String PRODUCT_CREATED = "A new product has been created: ";
+
+    @Override
+    public void sendNotification(NotificationDto dto) {
+        String message = PRODUCT_CREATED + dto.getPayload();
+        String targetQueue = (dto.getType() == TELEGRAM)
```

Comment:
apply strategy logic for notification channel types - because your code now is duplicated